docs/web/docs/4-faq/index.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: FAQ
+---
+
+### Can I use IAMbic to track changes in my cloud environment without managing permissions?
+
+Yes! IAMbic can be used to track changes to identities and permissions in your cloud environment,
+and import them in to Git. This provides a Git history into changes over time, and provides visibility into what is being changed,
+without needing to manually track changes in a spreadsheet or other tool.
+
+
+### If I use IAMbic to manage cloud identities, does IAMbic requre control over ALL of them?
+
+Certainly not! You are free to continue using other tools like Terraform, ClickOps, etc., for managing your cloud identities,
+while employing IAMbic to handle only a portion of them. Ideal initial candidates for IAMbic management include:
+
+1) Service Control Policies
+2) Cookie-cutter AWS Roles/Permission sensitive (Such as your `engineering` role that is used by all engineers)
+3) AWS Managed Policies
+4) Sensitive Applications or groups in Okta, Azure AD, or Google Workspace
+
+Regardless of whether you manage them via IAMbic or not, IAMbic will consistently track changes to all your identities.
+The fundamental principle here is storing all your IAM in one place, in a human-readable, bi-directional format. IAMbic gives you a
+nified view in Git instead of logging into each cloud console or using a different tool for each cloud provider.
+
+### How does IAMbic work in a production environment?
+
+IAMbic provides all essential tools and Github Actions to facilitate quick setup in a production environment. These Github actions execute the following tasks:
```

Comment:
This sentence is below as well

These Github actions execute the following tasks: